lib/css/components/_stacks-navigation.less

```diff
@@ -31,12 +31,20 @@
 
     .s-navigation--item {
         padding: @su6 @su12;
+        border: none;
```

Comment:
We are _so_ close to being able to use [all](https://caniuse.com/css-all) here. We would have to remove support for Edge 18, but I would much prefer that to all these resets. Perhaps I'll turn this into a mixin in the meantime.

```
.s-navigation--item {
    all: unset;
}
```